Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/TransformTool.java

```diff
@@ -319,12 +319,8 @@ private boolean areResizeBordersValid() {
 				&& resizeBoundHeightYBottom == workspace.getHeight() - 1) {
 			return false;
 		}
-		if ((resizeBoundWidthXRight + 1 - resizeBoundWidthXLeft)
-				* (resizeBoundHeightYBottom + 1 - resizeBoundHeightYTop) > maximumBoxResolution) {
-			return false;
-		}
-
-		return true;
+		return !((resizeBoundWidthXRight + 1 - resizeBoundWidthXLeft)
+				* (resizeBoundHeightYBottom + 1 - resizeBoundHeightYTop) > maximumBoxResolution);
```

Comment:
Please revert all changes to this file

Reply:
@geolas94 please revert all changes to this file